FutureNorthants/northants-website
# Update to Drupal 9.3.2 and add revision support to embedded paragraphs

### Testing

Everything 😭 

For the revision support of embedded paragraphs you need to:
1. Edit a page with a slice on it (e.g. a call to action button), change some aspect of the slice and save (but don't publish)
2. Refresh the page in the frontend and verify the content hasn't changed. 
3. Now publish the page in the CMS and verify the content has changed in the frontend.
4. Now edit the slice again and change some aspect, save as draft and verify hasn't changed in the frontend.
